### ISSUE TITLE
"Link to a board / card" doesn't work

### ISSUE THREAD

**Issue by nidico** [nextcloud/deck#3493]
**Bug description**

Trying to link a card to a board or to another card doesn't have any effect.

**To Reproduce**

1. Select an existing card
2. Click on 'Add to a project' in the 'Details' tab
3. Click on 'Link to a board'
4. Select an existing board and click on 'Select board' button

Alternatively,

3. Click on 'Link to a card'
4. Select an existing board and then an existing card and click on 'Link to card'

**Expected behavior**

The linked board / card should appear in the Details tab below, similar to what happens when 'Link to a file' is used.

**Client details:**
 - OS: Linux
 - Browser Firefox
 - Version 95
 - Device: Desktop

<details>
<summary>Server details</summary>

**Operating system**: Debian

**Web server:** Apache with nginx reverse proxy

**Database:** Postgres 13

**PHP version:** 7.4

**Nextcloud version:** 23.0.0

**Where did you install Nextcloud from:** Sources

**Signing status:** Don't know.

```
Login as admin user into your Nextcloud and access
http://example.com/index.php/settings/integrity/failed
paste the results here.
```

No errors have been found.

**List of activated apps:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your Nextcloud installation folder
```

Enabled:
  - accessibility: 1.9.0
  - activity: 2.15.0
  - bruteforcesettings: 2.3.0
  - calendar: 3.0.2
  - circles: 23.0.0
  - cloud_federation_api: 1.6.0
  - collectives: 0.17.22
  - comments: 1.13.0
  - contacts: 4.0.7
  - contactsinteraction: 1.4.0
  - dashboard: 7.3.0
  - dav: 1.21.0
  - deck: 1.6.0
  - federatedfilesharing: 1.13.0
  - federation: 1.13.0
  - files: 1.18.0
  - files_pdfviewer: 2.4.0
  - files_rightclick: 1.2.0
  - files_sharing: 1.15.0
  - files_trashbin: 1.13.0
  - files_versions: 1.16.0
  - files_videoplayer: 1.12.0
  - firstrunwizard: 2.12.0
  - logreader: 2.8.0
  - lookup_server_connector: 1.11.0
  - nextcloud_announcements: 1.12.0
  - notifications: 2.11.1
  - oauth2: 1.11.0
  - password_policy: 1.13.0
  - photos: 1.5.0
  - privacy: 1.7.0
  - provisioning_api: 1.13.0
  - recommendations: 1.2.0
  - richdocuments: 5.0.1
  - richdocumentscode: 21.11.6
  - serverinfo: 1.13.0
  - settings: 1.5.0
  - sharebymail: 1.13.0
  - support: 1.6.0
  - survey_client: 1.11.0
  - systemtags: 1.13.0
  - text: 3.4.0
  - theming: 1.14.0
  - twofactor_backupcodes: 1.12.0
  - updatenotification: 1.13.0
  - user_status: 1.3.1
  - viewer: 1.7.0
  - weather_status: 1.3.0
  - workflowengine: 2.5.0
Disabled:
  - admin_audit
  - encryption
  - files_external
  - mail
  - spreed
  - tasks
  - user_ldap

**Comment by marado** [comment]
I'm seeing this also, Deck 1.6.0. It worked in the past (but I can't confirm exactly in which version did it stop working).

**Comment by Poldi1977** [comment]
I can confirm this, I have the same problem here.

**Comment by LeoColman** [comment]
I've tried to track this one a bit.

I found `init-collections`, which contains a bit of the interaction
https://github.com/nextcloud/deck/blob/9bbef1fe4280b410ee8247036e11b4f0bff4986e/src/init-collections.js


`Dashboard.vue` also seems to have a bit of that
https://github.com/nextcloud/deck/blob/9bbef1fe4280b410ee8247036e11b4f0bff4986e/src/views/Dashboard.vue#L52

I think it was first introduced in commit https://github.com/nextcloud/deck/commit/0335229f4fa981a4409284cd19e1ef3e99eeed50

**Comment by Poldi1977** [comment]
The bug seems to have been fixed in the current version. In any case, it works again for me.

**Comment by nidico** [comment]
Confirmed that this is fixed in Deck 1.6.1, closing, thanks.
